system/nxplayer/nxplayer_mp3.c

```diff
@@ -22,12 +22,14 @@
  * Included Files
  ****************************************************************************/
 
-#include <sys/types.h>
+#include <nuttx/config.h>
```

Comment:
Not sure about this change as sys/types.h includes nuttx/config.h as the first line

Reply:
`sys/types.h` is a C library header, so it should be grouped with the others.
`nuttx/config.h` is just for standard practice in NuttX to be always included first.

Reply:
Ok.